src/pages/registration/contributors_tab/components/AuthorCard.tsx

```diff
@@ -123,23 +153,36 @@ const AuthorCard = ({ author, onMoveAuthor, onRemoveAuthorClick, openContributor
               <Tooltip title={t<string>('contributors.known_author_identity')}>
                 <StyledCheckIcon />
               </Tooltip>
-              <StyledTypography variant="body2">{t('contributors.verified_author')}</StyledTypography>
+              <StyledTypography variant="body2">{t('contributors.verified')}</StyledTypography>
             </>
           ) : (
-            <>
-              <Tooltip title={t<string>('contributors.unknown_author_identity')}>
-                <StyledWarningIcon />
-              </Tooltip>
-              <StyledVerifiedButton
-                color="primary"
-                data-testid={`button-set-unverified-contributor-${author.identity.name}`}
-                onClick={() => openContributorModal({ name: author.identity.name, index })}>
-                {t('contributors.verify_author')}
-              </StyledVerifiedButton>
-            </>
+            <StyledVerifiedButton
+              color="primary"
+              startIcon={
+                <Tooltip title={t<string>('contributors.unknown_author_identity')}>
+                  <WarningIcon />
+                </Tooltip>
+              }
+              variant="outlined"
+              data-testid={`button-set-unverified-contributor-${author.identity.name}`}
+              onClick={() => openContributorModal({ name: author.identity.name, index })}>
+              {t('contributors.verify_person')}
+            </StyledVerifiedButton>
           )}
         </StyledVerifiedSection>
         <StyledRightAlignedWrapper>
+          <StyledArrowSection>
+            {index >= 0 && index < contributorsLength && (
+              <StyledArrowButton color="secondary" onClick={() => onArrowMove('down')}>
+                <ArrowDownwardIcon />
+              </StyledArrowButton>
+            )}
+            {index !== 0 && index <= contributorsLength && (
```

Comment:
kan ikke denne forenkles litt? feks
```suggestion
            {index !== contributorsLength && (
              <StyledArrowButton color="secondary" onClick={() => onArrowMove('down')}>
                <ArrowDownwardIcon />
              </StyledArrowButton>
            )}
            {index !== 0 && (
```

Reply:
kan muligens forenkles ja - skal se på det